_episodes/format.md

```diff
@@ -3,51 +3,260 @@ title: "File format"
 teaching: 0
 exercises: 0
 questions:
-- "Key question (FIXME)"
+- "What is a file format?"
+- "What file formats should I use?"
 objectives:
-- "First learning objective. (FIXME)"
+- "Name common formats used in climate-related domains."
+- "Understand different types of data."
+- "Choose correct file formats for your work."
+
 keypoints:
-- "First key point. Brief Answer to questions. (FIXME)"
+- "Choose formats that are common to your field/community to ensure the interoperability and reusability of your data."
+- "Make sure that the file formats you choose can hold the necessary data elements and information."
+- "Decide on how long do you intend to preserve your data."
+- "Make sure to check requirements of the repository where data is stored."
 ---
 
+## File format
+
+A [file format](https://en.wikipedia.org/wiki/File_format) is the structure of a file.
+It determines how the data within the file is organized.
+A File format is also called a file extension.
+Files are usually named as `filename.format`.
+For example, climate.jpeg is a file named climate with [jpeg](https://en.wikipedia.org/wiki/JPEG) format.
+You might know that `jpeg` is a commonly used format for digital images.
+Therefore, the file `climate` is an image.
+
+## Common file format in climate-related domains
+
+In climate-related domains e.g. weather and climate science, Earth observation science, hydrology,
+data can be in many types and for different purposes.
+In this section, we will learn about the common formats of data.
```

Comment:
```suggestion
In this section, we will explore some common data formats.
```
I'd be careful with stating that 'we will learn'. Personally I feel that sounds a bit derogatory.

Reply:
> Hey @SarahAlidoost thanks for adding this, that's a great start!
> 
> I have a number of suggestions, mostly small. Overall, I wonder if it would be better to move some of the exercises about choosing a data format and assessing whether it helps to make the data FAIR, more to the end of the episode. Now you're first doing this (with the implicit assumption that you're choosing this format to _publish_ the data), and then at the end you state that different formats may be more suitable during the evaluation phase e.g.

@Peter9192 thank you for the comments, very helpful. The text is revised now. Please see my comments as well. 

The exercises are related to its own section especially those **bold** text in the section. The approach that I took is as: first, some material is introduced, next, the user is challenged. Therefore, it does not make sense to move them to the end of the episode. To address your concern, we can remove them. Also, see my comment on acceptable and recommended formats.

I am also thinking about adding a nice figure to this lesson. I will make a new PR when I found something.